__main__.py

```diff
@@ -97,7 +108,7 @@ def on_event(event):
                         conversation))
 
     def show_event(event):
-        return lambda: print(event.text)
+        return lambda: logger.info('Received message: %s', event.text)
```

Comment:
This also logs messages we send as "received", which while technically correct, seems a little unintuitive.